### ISSUE TITLE
Error when enabling Blender add-on

### ISSUE THREAD

**Issue by TJScalzo** [TheSignPainter98/adjust-keys#63]
### Describe the Bug

When attempting to enable the add-on in Blender preferences, an error is thrown that prevents it from being enabled.

### To Reproduce

1. Download .zip from the [release](https://github.com/TheSignPainter98/adjust-keys/releases/tag/v1.4.0) page.
2. Install add-on in Blender by selecting the .zip file.
3. Click the checkbox to enable the add-on.
4. See that an error report pops up instead:

```
Traceback (most recent call last):
  File "/Applications/Blender.app/Contents/Resources/2.82/scripts/modules/addon_utils.py", line 382, in enable
    mod.register()
  File "/Users/username/Library/Application Support/Blender/2.82/scripts/addons/adjust_keys_blender_addon/__init__.py", line 319, in register
    register_class(KczaCustomPropertyGroup)
ValueError: bpy_struct "KczaCustomPropertyGroup" registration error: verbosity could not register
```

### Expected Behavior

The add-on would be enabled properly and could be used in Blender.

### Environment

- OS: macOS Catalina version 10.15.7
- Script: Adjust-Keys Blender Addon

### Additional Context

I made sure I had Python installed and attempted to install/enable the add-on again. It threw the same error.

**Comment by TheSignPainter98** [comment]
Thanks for letting me know about this! I'll see what I can do, but I'm having trouble replicating the bug. What Blender version are you running?

**Comment by TJScalzo** [comment]
I'm running Version 2.82a (2.82a 2020-03-12). Do you think I should try updating Blender?

**Comment by TheSignPainter98** [comment]
Okay, I've replicated the bug in 2.82 and will just work on fixing it for that Blender version. I haven't had this on newer versions (2.83+) so I think updating should also get rid of the problem.

**Comment by TheSignPainter98** [comment]
I've applied the fix and tested Adjust-Keys on Blender 2.82 and 2.82a. The changes are now in the main branch (via #65) and will be in v1.5.0 which I plan to release in the next day or two.

I'll keep this thread open for a little while in case the issue persists.
